### PR TITLE
Issue204/support multiple units

### PR DESCRIPTION
**Pull request by mateuszkrasucki**
Closes: #204

### CODE REVIEW

File: frontend/local/src/containers/patients/detail/index.js [hunk 1]
```diff
@@ -90,26 +93,50 @@ class PatientDetail extends React.Component {
...
                                 <div className="col-sm-4">
                                     <h5>Height</h5>
-                                    {bodyMeasurements.height} cm
+                                    <ValueWithUnitString
+                                        valueUnit="cm"
+                                        stringUnit={this.props.lengthUnit}
+                                        precision={0}
+                                        formatting={SHORT}
+                                        value={bodyMeasurements.height}
+                                    />
```
Comment (dominikznidar) [inline frontend/local/src/containers/patients/detail/index.js]: We should try to get to the same interface with displaying values as with inputs. Patient details page should not be aware of the value unit set in config.

File: frontend/local/src/containers/settings/popover.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+const mapStateToProps = (state, ownProps) => {
+    return {
+        status: state.status.status
+    }
+}
+
+const mapDispatchToProps = dispatch => bindActionCreators({}, dispatch)
+
+export default connect(mapStateToProps, mapDispatchToProps)(SettingsPopover)
```
[the remark this answers is not in the document]
Comment (mateuszkrasucki) [reply frontend/local/src/containers/settings/popover.js]: removed

File: frontend/local/src/containers/settings/popover.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            </React.Fragment>
+        )
+    }
+}
+
+const mapStateToProps = (state, ownProps) => {
+    return {
+        status: state.status.status
+    }
+}
```
[the remark this answers is not in the document]
Comment (mateuszkrasucki) [reply frontend/local/src/containers/settings/popover.js]: removed

File: frontend/local/src/containers/patients/new/step3.js [hunk 4]
```diff
@@ -100,24 +106,61 @@ const renderBabyForm = ({ babyFoods, deliveryTypes, communicationTypes }) => (
...
+                                name="weightAtBirth"
+                                label="Weight at birth"
+                                placeholder="Weight"
+                                component={SimpleUnitInput}
+                                unit={GRAMS}
+                                precision={0}
+                                min={0}
+                                warn={babyHeightExpectedRange}
+                            />
+                        )}
```
[the remark this answers is not in the document]
Comment (mateuszkrasucki) [reply frontend/local/src/containers/patients/new/step3.js]: done

File: frontend/local/src/containers/patients/detail/index.js (hunk 1 above)
Comment (mateuszkrasucki) [reply frontend/local/src/containers/patients/detail/index.js]: done

**Pull request merged**